components/src/core/components/Button/button.scss

```diff
@@ -138,17 +138,9 @@
 
 /*Glass Button */
 .oxd-button {
-  .oxd-button-icon,
```

Comment:
Why are we changing the common button for this? What is the impact from this?

Reply:
Original styles for buttons has been as follows.

.oxd-button-icon {
  color: inherit;
  vertical-align: bottom;
  position: relative;
  left: -0.5rem;
}

It has been modified to satisfy the icons for button 3 months back and with the new changes we have change the styles further so the attributes such as color: inherit;, vertical-align: bottom and position: relative; are been removed. 

This can impact any other places which has buttons with icons including Buzz module icons. Can we avoid altering the button component and only add the styles for SelectButton component?

Reply:
Reverted changes done in this file